CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ if (CMAKE_TOOLCHAIN_FILE)
 else ()
   # By default, use gcc cross-compiler for ARM-Thumb
   set(CMAKE_C_COMPILER arm-linux-gnueabihf-gcc)
-  add_compile_options(-mthumb)
+  add_compile_options(-mthumb -g)
```

Comment:
unneeded :see_no_evil: , but we can keep, i used it to debug the launcher itself

Reply:
Ok, I drop it.